.semaphore/push_images.yml

```
@@ -2,7 +2,7 @@ version: v1.0
 name: Operator CD
 agent:
   machine:
-    type: e1-standard-4
+    type: f1-standard-4
```

I think this can actually be f1-standard-2, I think these machines are bigger. The 4 is apparently much larger, might want to check with @sujeet-kr though.

---

Yes `e1-standard-4`'s can be replaced with `f1-standard-2`s, if the reason for using `e1-standard-4` was memory and storage.
```suggestion
    type: f1-standard-2
```